src/request_handler.rs

```diff
@@ -241,6 +252,23 @@ pub(crate) mod tests {
     });
   }
 
+  #[test]
+  fn listing_is_sorted_alphabetically() {
+    test(|url, dir| async move {
+      let www = dir.join("www");
+      std::fs::write(www.join("b"), "").unwrap();
+      std::fs::write(www.join("c"), "").unwrap();
+      std::fs::write(www.join("a"), "").unwrap();
```

Comment:
The order of read directory entries is both platform and filesystem dependent. (See [here](https://doc.rust-lang.org/std/fs/fn.read_dir.html#platform-specific-behavior). This is the documentation for `std::fs::read_dir`, but I think it applies to `tokio::fs::read_dir` as well.) This test (with files `b`, `c` & `a`, written in that order) seems to fail consistently on my machine when not sorting the output of `read_dir`. (We could try to come up with a more robust test with e.g. [`proptest`](https://crates.io/crates/proptest), but I think that's not necessary.)

Reply:
I'm fine without a more robust test.